db/migrate/20170331150112_remove_smart_from_vms.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveSmartFromVms < ActiveRecord::Migration[5.0]
+  def change
+    remove_column :vms, :smart, :boolean
```

Comment:
Will also want to remove the caller: [app/models/vm_or_template.rb#L315](https://github.com/ManageIQ/manageiq/blob/43bfce2b0d6d8a743740f29be2c9c1d5a704aebf/app/models/vm_or_template.rb#L315)